assets/scss/application.scss

```diff
@@ -30,7 +30,7 @@ $govuk-page-width: $moj-page-width;
 }
 
 .govuk-panel--warning {
-  background-color: #FAC710;
+  background-color: #fac710;
```

Comment:
A very minor change obviously but I don't _think_ this is needed as part of the upgrade? Guessing it was a style thing you noticed in passing? I'd put this in another commit so it's separate from the riskier change should that part need to be reverted for whatever reason, you're other work will persist 👍

Reply:
hmmm very strange, I didn't change this at all. It must have happened automatically somehow. Anyway I will remove it, thanks